src/core/QRSVG.ts

```diff
@@ -48,6 +48,7 @@ export default class QRSVG {
     this._element = document.createElementNS("http://www.w3.org/2000/svg", "svg");
     this._element.setAttribute("width", String(options.width));
     this._element.setAttribute("height", String(options.height));
+    this._element.setAttribute("viewBox", `0 0 ${options.width} ${options.height}`);
```

Comment:
do we need to do a search all for `this._element.setAttribute` and add `viewBox` also? There are three places similar code appear
<img width="421" alt="image" src="https://github.com/user-attachments/assets/4b02644c-3754-40a8-b916-81a9ca1688f8">